src/main/java/ch/njol/skript/conditions/CondEntityStorageIsFull.java

```diff
@@ -0,0 +1,68 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.EntityBlockStorage;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Entity Storage Is Full")
+@Description("Checks to see if an entity block storage (i.e beehive) is full.")
+@Examples({
+	"if the entity block storage of {_beehive} is full:",
+		"\trelease the entity block storage of {_beehive}"
+})
+@Since("INSERT VERSION")
+public class CondEntityStorageIsFull extends Condition {
+
+	static {
+		Skript.registerCondition(CondEntityStorageIsFull.class, ConditionType.PROPERTY,
+			"[the] entity block storage of %blocks% (is|are) full",
+			"[the] entity block storage of %blocks% (isn't|is not|aren't|are not) full");
+	}
```

Comment:
i'm not a big fan of this syntax, it feels rather awkward
maybe `if %blocks% is/are full of entities`?